WcaOnRails/spec/requests/api_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "API mics" do
+  describe "GET #records" do
+    let!(:wr333) { FactoryBot.create :result, eventId: "333", best: 444, countryId: "Australia" }
+    let!(:nr333) { FactoryBot.create :result, eventId: "333", best: 555, countryId: "USA" }
+    let!(:dnf444) { FactoryBot.create :result, eventId: "444", best: SolveTime::DNF_VALUE, average: SolveTime::DNF_VALUE, countryId: "USA" }
+
+    it "renders current records", clean_db_with_truncation: true do
+      # Compute necessary data.
+      AuxiliaryDataComputation.compute_concise_results
+      Timestamp.create(name: "compute_auxiliary_data_end", date: DateTime.now)
```

Comment:
Nice! It's unfortunate that you have to create the timestamp here. Would it be possible to invoke `ComputeAuxiliaryData.perform` directly here instead?

Reply:
That would compute all the other tables as well, and as it already consumes relatively long time, I'd avoid adding into it, does that make sense?

Reply:
Ah, bummer. I would expect CAD computation to be fairly fast with only 3 results in the database. What's the speed difference like?

Reply:
Well, not a big one:

![image](https://user-images.githubusercontent.com/17034772/35705204-103cff2a-07a2-11e8-8c1d-372106f54c09.png)

I was intuitively optimising it as our test suit takes a bunch of time already. I can change this

Reply:
Wow, 1.3 seconds is more than I was expecting! Let's do whatever you think is best. I know that 1 second here or there can add up to a lot over time.